### PR TITLE
Create ci-cd.yml

### PR DESCRIPTION
**Pull request by killerninja8**

### CODE REVIEW

**Comment by killerninja8** [comment]
this is slightly diff from tiff's version fyi

File: .github/workflows/ci-cd.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
+name: ci-cd
+
+on:
+  push:
+    branches:
+      - master
```
Comment (scout-mckee) [inline .github/workflows/ci-cd.yml:6]: Should this be main instead of master?

File: .github/workflows/ci-cd.yml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  pull_request:
+    branches:
+      - master
```
Comment (scout-mckee) [inline .github/workflows/ci-cd.yml:9]: for this as well- should this be main?

**Review by scout-mckee — CHANGES_REQUESTED** [review]
I am not sure if this should be main or master? sorry I am still getting used to these workflows. If it is master then it looks good to merge

File: .github/workflows/ci-cd.yml (hunk 1 above)
Comment (killerninja8) [reply .github/workflows/ci-cd.yml:6]: yes you're right, i'll change that!

Comment (killerninja8) [inline .github/workflows/ci-cd.yml:6]:
```suggestion
      - main
```

File: .github/workflows/ci-cd.yml (hunk 2 above)
Comment (killerninja8) [reply .github/workflows/ci-cd.yml:9]: this as well! thanks for pointing that out

Comment (killerninja8) [reply .github/workflows/ci-cd.yml:9]:
```suggestion
      - main
```

**Review by scout-mckee — APPROVED** [review]

**Review by scout-mckee — APPROVED** [review]

**Pull request merged**